common/app/views/fragments/items/facia_cards/contentCard.scala.html

```diff
@@ -58,7 +58,7 @@
             }
         }
 
-        @if(item.contentType.name.toLowerCase == "audio") {
+        @if(item.designType == "audio") {
```

Comment:
I think we also have a conditional check for `gallery` in master

Reply:
OK good spot, I'll rebase and update that.